services/activitylog/pkg/service/service.go

```diff
@@ -265,70 +376,113 @@ func (a *ActivitylogService) activities(rid *provider.ResourceId) ([]RawActivity
 	}
 
 	var activities []RawActivity
-	if err := json.Unmarshal(records[0].Value, &activities); err != nil {
-		return nil, fmt.Errorf("could not unmarshal activities: %w", err)
+	if err := msgpack.Unmarshal(records[0].Value, &activities); err != nil {
```

Comment:
Nit: Hm, somehow I think we should have a debug log statement here when falling back json. (Feel free to ignore me here)